packages/internet-header/src/components/post-internet-header/post-internet-header.tsx

```diff
@@ -505,6 +511,11 @@ export class PostInternetHeader {
               </If>
             </post-main-navigation>
             <div class="main-navigation-controls">
+              <If condition={!this.isMainSlotEmpty}>
+                <div class="main-navigation-custom-content">
+                  <slot name="main" onSlotchange={e => this.handleMainSlotChange(e)}></slot>
+                </div>
+              </If>
```

Comment:
This seems suspicious, because it won't render the slot if it's empty, but how do you know it's empty if it's not there? While it's working with the current implementation, I'm not sure what will happen if the content comes from Angular, meaning it's rendered client side and maybe after the header initialized.